src/test/resources/testenv.properties

```diff
@@ -0,0 +1,6 @@
+#browser to be used for testing
+browser = chrome
+#Base URL of the website
+baseurl = http://opensource.demo.orangehrmlive.com
+#Location of screenshots to store
+screenshotpath = D:\\Projects\\JavaPractice\\out\\screenshots
```

Comment:
please use relative paths only.

Reply:
Accepted and Fixed.
Since screenshots are captured in the reports, storing screenshots to a folder is not considered.